sdks/python/apache_beam/runners/common.py

```diff
@@ -716,7 +717,7 @@ def _reraise_augmented(self, exn):
           traceback.format_exception_only(type(exn), exn)[-1].strip()
           + step_annotation)
       new_exn._tagged_with_step = True
-    raise_(type(new_exn), new_exn, original_traceback)
+    raise_with_traceback(new_exn)
```

Comment:
This was changed since `raise_` creates a new exception object, which removes the assigned attributes.